client/settings/settings.tsx

```diff
@@ -223,6 +223,7 @@ const NavEntryRoot = styled.div<{ $isActive: boolean }>`
   align-items: center;
 
   border-radius: 4px;
+  overflow: hidden;
```

Comment:
Is this really desirable?

Reply:
It's the only way I've found on how to have the border-radius show up on the child Ripple component. Maybe there's a better way to make child components inherit border-radius from their parents?

Reply:
`contain: content` (or one of the other options) will likely work